dotnet/Program.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using io.fusionauth;
+using io.fusionauth.domain;
+using io.fusionauth.domain.api;
+using io.fusionauth.domain.api.user;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using io.fusionauth.domain.oauth2;
+using io.fusionauth.domain.search;
+
+namespace Setup
+{
+    class Program
+    {
+        private static readonly string apiKey = Environment.GetEnvironmentVariable("fusionauth_api_key");
+        private static readonly string fusionauthURL = "http://localhost:9011";
+
+        private static readonly string applicationId = "4243b56f-0b45-4882-aa23-ac75eea22d22";

```

Comment:
Let's use the standard applicationid: 
```suggestion
        private static readonly string applicationId = "e9fdb985-9173-4e01-9d73-ac2d60d1dc8e";

```